code/modules/research/xenobiology/crossbreeding/selfsustaining.dm

```diff
@@ -23,11 +24,11 @@ Self-sustaining extracts:
 		A.extract = X
 		A.icon = icon
 		A.icon_state = icon_state
-		A.color = color
+		A.color = auto_color
```

Comment:
Change this back, hardcoding the colours is not elegant and shouldn't be necessary now you've fixed the runtime

Reply:
Thanks for the help, I hardcoded it due to that line being what was originally causing the problem, but I guess it just works now? it might have been that the name for the autoslime was originally assigned in its Initialization(). The only problem now is that the name comes out as "self-sustaining red slime extract"